Student_BCOs/Updated_Nirmala_Ramnarine_BCO_000537.json

```diff
@@ -0,0 +1,563 @@
+                "orcid": "",
+                "affiliation": " Center for Engineering in Medicine, Harvard Medical School, Boston, MA 02114, USA; Department of Surgery, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "David T. Miyamoto",
+                "orcid": "https://orcid.org/0000-0003-3692-8823",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Radiation Oncology, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Nicola Aceto",
+                "orcid": "https://orcid.org/0000-0001-9579-6918",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": " Francesca Bersani",
+                "orcid": "https://orcid.org/0000-0002-0343-7078",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Brian W. Brannigan",
+                "orcid": "",
+                "affiliation": " Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Kristina Xega",
+                "orcid": "",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Jordan C. Ciciliano",
+                "orcid": "https://orcid.org/0000-0003-1427-2608",
+                "affiliation": "Massachusetts General Hospital Cancer Center, Harvard Medical School, Boston, MA 02114, USA; Department of Medicine, Harvard Medical School, Boston, MA 02114, USA.",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy" , each publication author should also have the "derivedFrom" contribution type. Please add. See #112